tests/unit/ValidationUtilsTest.js

```diff
@@ -71,9 +71,9 @@ describe('ValidationUtils', () => {
             expect(ValidationUtils.isValidWebsite('192.168.0.1')).toBe(false);
         });
 
-        test('Invalid URLs without protocols', () => {
-            expect(ValidationUtils.isValidWebsite('www.expensify.com')).toBe(false);
-            expect(ValidationUtils.isValidWebsite('expensify.com')).toBe(false);
+        test('Valid URLs without protocols', () => {
```

Comment:
We need websites with a valid protocol, so this is invalid.

Reply:
http, https, ftp are valid protocols accepted by the server.

Reply:
I think yes, when setting up a bank account. In fact we have that validation currently.